deploy/aws/eks/manifests/crd.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Can we symlink this instead of copying?

Reply:
No, this module is supposed to be workable outside this repository. But it is definitely not a good idea to keep a copy here.

How about `kubectl appy https://raw.githubusercontent.com/pingcap/tidb-operator/v1.0.0-beta.3/manifests/crd.yaml`?

Reply:
Why does this need to work outside this repository? I have been referencing manifests with symlinks in the GCP terraform. The deployment doesn't work without a good way to reference our charts and manifests.

The suggested update is problematic because it will fall out of sync. Better would be to have a variable "manifests" which could be set to `../../manifests` or `https://raw.githubusercontent.com/pingcap/tidb-operator/v1.0.0-beta.3`.

Does it help to move the manifests directory under the deploy directory?

Reply:
> Why does this need to work outside this repository? I have been referencing manifests with symlinks in the GCP terraform. The deployment doesn't work without a good way to reference our charts and manifests.

Cases that use these modules outside this repository:
- terraform store `state` in the working dir, users may copy this module to manage different EKS instances (one of our colleges have tried this and got hurt by the symlink)
- for advanced users, it is possible to compose the `tidb-operator` module and `tidb-cluster` module into their own terraform scripts. A self-contained module will be easier to be composed in this case

> Does it help to move the manifests directory under the deploy directory?

Yes, to some extent. But the `manifest` directory is still necessary because the `ebs-gp2` storage class and `local-volume-provisioner.yaml` are dedicated to AWS. I've considered use a small overlay yaml to customize the base `local-volume-provisioner.yaml` via kusomize, but this has a low priority.

Reply:
I used kustomize for GCP, it was quite easy: https://github.com/pingcap/tidb-operator/tree/master/deploy/gcp/manifests/local-ssd

Reply:
With respect to state, it seems that users are solving a problem by copying and in response we are copying :)
I want to get away from copying as a solution to anything. The way I solve this is by having users instantiate a module, rather than just changing a terraform variable files. A user instantiates the module in a directory representing their environment (staging, production). The usability is about the same for one instantiation, but when you do multiple everything is much better.

Reply:
I think in our case we are missing a nice mechanism to depend on our manifest files. The only way I can see how to do it is to use the `file` function to read the file in (and we can write it out to a new local file to avoid memory consumption). But then I think we still end up needing the entire git repo, so it doesn't really seem better than a symlink.